ci/build/build-pipeline.yaml

```diff
@@ -199,6 +217,7 @@ spec:
     - name: build-vsp-image
       serial: true
       serial_groups: [build-vsp]
+      <<: *slack-notification
```

Comment:
The more I read about the `<<:` operator, the less convinced I am that it's necessary. 

`<<:` seems to perform a merge, so it's useful when you want to add some key-value-pairs from an anchor, but without overwriting existing keys. We don't have any existing keys, so it isn't to perform a merge necessary. 

a `*` should be enough.

That being said, what you have will work as intended.

Reply:
Agree, I've pushed a change to remove the << chevrons